packages/docs-reanimated/static/examples/FABButton.js

```diff
@@ -0,0 +1,156 @@
+import React from 'react';
+import { StyleSheet, SafeAreaView, View, Pressable, Text } from 'react-native';
+import Animated, {
+  withDelay,
+  interpolate,
+  useAnimatedStyle,
+  useSharedValue,
+  withSpring,
+  withTiming,
+} from 'react-native-reanimated';
+
+const AnimatedPressable = Animated.createAnimatedComponent(Pressable);
+
+const SPRING_CONFIG = {
+  duration: 1200,
+  overshootClamping: true,
+  dampingRatio: 0.8,
+};
+
+const OFFSET = 60;
+
+const FABButton = ({ animatedStyles, buttonLetter }) => (
```

Comment:
```suggestion
const FloatingActionButton = ({ animatedStyles, buttonLetter }) => (
```
Let's use the full name as `FABButton` expands to `Floating Action Button Button` 😄

Reply:
😭 😆